recipes/jolt-physics/recipe.yaml

```
@@ -0,0 +1,62 @@
+context:
+  version: 5.0.0
+
+package:
+  name: jolt-physics
+  version: ${{ version }}
+
+source:
+  url: https://github.com/jrouwe/JoltPhysics/archive/refs/tags/v${{ version }}.zip
+  sha256: 2c98750316d565d553cb2c15d494b6b2ea4fbe55eb2ca18df536c9f4ac340d36
+
+requirements: 
+  build: 
+    - ${{ compiler("cxx") }}
```

Don't we need the `stdlib` function here too?

```suggestion
    - ${{ stdlib("c") }}
```

---

```suggestion
    - ${{ compiler("cxx") }}
    - ${{ stdlib("c") }}
```